### ISSUE TITLE
Epochs are not being registered (It doesn't resume and save the val_acc as shown in the Image)

### ISSUE THREAD

**Issue by alikazi88** [ItamarRocha/binary-bot#23]
I have gone through the model.fit snippet and everthing seems to be fine but in vain also training.py returns no output is something i am missing here ? please help !!!!

**Comment by samchin89** [comment]
Run “testing.py”.

**Comment by alikazi88** [comment]
Yup was doing the same thing again again still epochs start from 0

**Comment by samchin89** [comment]
What error printed? 
I am using tensorflow 2.3.0 with keras 2.2.0 

other tensorflow version renders me problems.

**Comment by traderpedroso** [comment]
> I have gone through the model.fit snippet and everthing seems to be fine but in vain also training.py returns no output is something i am missing here ? please help !!!!

change val_acc to val_accuracy

**Comment by ericleonardo** [comment]
> > I have gone through the model.fit snippet and everthing seems to be fine but in vain also training.py returns no output is something i am missing here ? please help !!!!
> 
> change val_acc to val_accuracy

Oi Emerson @traderpedroso , vc continua usando este bot do Itamar? Qual acertividade (win ratio) vc consegue fazer com ele? Instalei agora e estou testando, mas após estabelecer conexão com a IQ Option, o código simplesmente não faz mais nada.. será que API mudou novamente e o bot fica aguardando os dados que não chegam? Obrigado

**Comment by traderpedroso** [comment]
> > > I have gone through the model.fit snippet and everthing seems to be fine but in vain also training.py returns no output is something i am missing here ? please help !!!!
> > 
> > 
> > change val_acc to val_accuracy
> 
> Oi Emerson @traderpedroso , vc continua usando este bot do Itamar? Qual acertividade (win ratio) vc consegue fazer com ele? Instalei agora e estou testando, mas após estabelecer conexão com a IQ Option, o código simplesmente não faz mais nada.. será que API mudou novamente e o bot fica aguardando os dados que não chegam? Obrigado

usa esta api simplificada e atualizada [API IQ](https://github.com/ejtraderLabs/ejtraderIQ)
